src/www/ui/api/Models/Reuser.php

```diff
@@ -81,25 +81,25 @@ public function __construct($reuseUpload, $reuseGroup, $reuseMain = false,
    * @throws \UnexpectedValueException If reuse upload of reuse group are non
    * integers
    */
-  public function setUsingArray($reuserArray)
+  public function setUsingArray($reuserArray, $version = ApiVersion::V1)
```

Comment:
Let's make sure the API V1 is the default, in the implementations of this method.